AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -491,10 +491,17 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
     spec = [ASInsetLayoutSpec insetLayoutSpecWithInsets:contentEdgeInsets child:spec];
   }
   
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
-    stack.sizeRange = ASRelativeSizeRangeMakeWithExactCGSize(self.preferredFrameSize);
+#if DEBUG
```

Comment:
So you are on the side to remove all usages in e.g. `ASTextNode`, `ASButtonNode`, `ASImageNode` ... the "problem" with that is just that this would break a lot of code and `preferredFrameSize` would basically be useless and we could remove it in the first place. I think it would make even more sense to remove it at all if we decide to not gracefully remove it in ASDK code as than developer would get an error and have to fix the places instead of just a warning or nothing if they don't have Xcode warnings enabled for deprecation.